src/components/CalcScreen.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { Textfit } from 'react-textfit';
+import { CalcContext } from '../context/CalcContext';
 
 class CalcScreen extends React.PureComponent {
   render() {
-    const { value } = this.props;
+    const { calc } = this.context;
     return (
-      <div className="calcScreen">
-        {value}
-      </div>
+      <Textfit className="calcScreen" max={50} mode="single">
+        {calc.num ? calc.num : calc.res}
+      </Textfit>
     );
```

Comment:
- [ ] Great job so far 👏🏻 However, according to the project requirements. To make work easy and concentrate on how to work with React events, you are advised to use the already developed `math logic` needed to make work the Calculator component. Let's make sure that:
*  You hava an `operate.js` file copied from [this one](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js).
* You also have a `calculate.js` file copied from [this one](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js)
- You need to create a `logic` forder directory for the  `operate.js` and the `calculate.js` files.

Reply:
Thanks Mr. @BenMukebo  for the great feedback. I have already implemented the requested changes.